<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Epayment</class>
 <widget class="QMainWindow" name="Epayment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(136, 202, 94);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>430</y>
      <width>811</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(136, 202, 94);
font: 75 10pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Â© 2024 LMMN, Inc. All rights reserved</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="back_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Sans Serif&quot;;
border-radius: 10px; 
background-color: rgb(255, 255, 255);
color: rgb(136, 202, 94);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 19pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Payment Method: E-Cash</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Please scan the provided QR code below</string>
    </property>
   </widget>
   <widget class="QFrame" name="QRbitMapFrame">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="QRBitMap">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>231</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Total Amount to be Paid</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalPrice">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="proceed_Button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>390</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 75 12pt &quot;MS Sans Serif&quot;;
border-radius: 10px; 
background-color: rgb(255, 255, 255);
color: rgb(136, 202, 94);</string>
    </property>
    <property name="text">
     <string>Proceed</string>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>frame</zorder>
   <zorder>back_Button</zorder>
   <zorder>label_2</zorder>
   <zorder>QRbitMapFrame</zorder>
   <zorder>frame_3</zorder>
   <zorder>proceed_Button</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
